tests/support/fixtures.ts

```diff
@@ -87,8 +92,7 @@ export const test = base.extend<{
           route.request().url().startsWith("https://esm.sh/vscode-oniguruma@1")
         ) {
           route.fulfill({
-            status: 200,
-            path: "./tests/support/onig.wasm",
+            path: onigWASMPath,
```

Comment:
Do we even need any of this in the test file? Should it not just work now that we load it in the `syntax.ts` file?

Reply:
Yeah you're right, we don't need this anymore.
8551ab4 removes it